Add O lettered article stubs from glossary (#3859)
ppy/osu-wiki: wiki/Disambiguation/Ouendan/en.md

> ---
> stub: true
> ---

Reviewer: ```suggestion
---
stub: true
tags:
  - Osu! Tatake! Ouendan
---
```

# Ouendan

Ouendan has three meanings:

> - In Japanese culture, a cheer squad which, similar to western cheerleaders, rallies a sports team on using drums, horns, banners and yells of encouragement.

Reviewer: ```suggestion
- In Japanese culture, a cheer squad which, similar to western cheerleaders, rallies a sports team using drums, horns, banners and cheers of encouragement.
```
could be misread

> - The pair of Japanese DS games that osu! is based on.
>   This term may sometimes be used to refer specifically to the first of these games.

Reviewer: ```suggestion
- The pair of Japanese DS games that osu! is based on.
  - This term may sometimes be used to refer specifically to the first of these games.
```
